MaterialLib/Fluid/Viscosity/VogelsLiquidDynamicViscosity.h

```diff
@@ -0,0 +1,111 @@
+/**
+ *  \brief Declaration of class for the pressure dependent viscosity model.
+ *
+ *  \copyright
+ *   Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ *  \file  VogelsLiquidDynamicViscosity.h
+ *
+ */
+
+#ifndef VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+#define VOGELS_LIQUID_DYNAMIC_VISCOSITY_H_
+
+#include <string>
+#include <vector>
+#include <array>
+#include <cmath>
+
+#include "MaterialLib/Fluid/FluidProperty.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/** A temperature dependent viscosity model.
+ * <a href="ddbst"> http://ddbonline.ddbst.de/VogelCalculation/VogelCalculationCGI.exe</a>
+ */
+template <typename VogelsConstants>
+class VogelsLiquidDynamicViscosity final : public FluidProperty
+{
+public:
+    /**
+     *  \brief Viscosity defined by \f$e^{A+\dfrac{B}{C+T}}\f$
+     *  \param constants Constants of the fluid.
+     *
+     */
+    explicit VogelsLiquidDynamicViscosity(const VogelsConstants& constants)
+        : _constants(constants)
+    {
+    }
+
+    /// Get model name.
+    std::string getName() const override
+    {
+        return "Liquid viscosity by Vogel's equation";
+    }
+
+    /** Get viscosity value.
+     *  \param var_vals Variable values in an array. The order of its elements
+     *                  is given in enum class PropertyVariableType.
+     */
+    double getValue(const ArrayType& var_vals) const override
+    {
+        const double T = var_vals[static_cast<int>(PropertyVariableType::T)];
+        return 1.e-3 *
```

Comment:
A constant for SI unit. Added it in the formulas documentation and added a comment about it above the the return.